doc/api/crypto.markdown

```diff
@@ -625,9 +625,17 @@ Asynchronous PBKDF2 function.  Applies the selected HMAC digest function
 salt and number of iterations.  The callback gets two arguments:
 `(err, derivedKey)`.
 
+The number of iterations passed to pbkdf2 should be as high as possible, the
+higher the number, the more secure it will be, but will take a longer amount of
+time to complete.
+
+Chosen salts should also be unique and random (ideally using crypto.randomBytes)
```

Comment:
It'd be great if `crypto.randomBytes` could be linked to the documentation for the function.